runtime/src/accounts_db.rs

```diff
@@ -2612,12 +3471,36 @@ impl AccountsDB {
         dead_slots
     }
 
-    fn clean_dead_slots(
+    fn clean_dead_slots_common(
```

Comment:
nits: I want to name this fn something neutral to be called from `purge_slot_cache_keys` and `clean_stored_dead_slots`. (Yeah, `clean` and `purge` verb is alreay confusing)..

how about `finalize_dead_slots_handling`?

Reply:
changed to `finalize_dead_slot_removal`